### DISCUSSION TITLE
Keeping it 💯 (Million Developers)  [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#45463] [New to GitHub]
GitHub’s year continues to be off to a good start, we hit 100 million GitHub users and this is only just the beginning! Celebrate with us by heading to your profile (or anyone else’s) for a festive animation 🟩. 

Members have been more helpful than ever, whether answering complex questions or providing awesome feature requests. Enjoy the latest and greatest in New to GitHub 😌. 

🌟**Community Superstars**: A round of applause for these community citizens 👏 
* @jge162: [Shared their wisdom](https://github.com/orgs/community/discussions/44343) 🦉 with a new GitHub user why you would add reviewers/collaborators to your repo. Thanks for the perspective!
* @rashidwassan: Compiled an [excellent list](https://github.com/orgs/community/discussions/44405) of steps to get you started on your GitHub journey. Their breakdown makes learning GitHub manageable and fun! 
* @KristofHorvath: [Needed clarity on collaborators and Pro plans](https://github.com/orgs/community/discussions/44864#discussion-4776894) which led to us realizing the language on our Compare Plans could be updated to be more easily understandable 🤓. Thank you for helping us improve and make GitHub better for everyone! 

🆒 **Cool stuff**™️ happening around GitHub and the Community
* **Copy an existing Project**: We've shipped the ability to copy your project's views, custom fields, and draft issues 👯. Leave your feedback with @2percentsilk and the Projects team [here](https://github.com/orgs/community/discussions/44815). 
* **Introducing the GitHub Bug Bounty swag store**: [Learn how](https://github.blog/2023-01-23-introducing-the-github-bug-bounty-swag-store/) you can get swag 🧢 and keep GitHub safe! 
* **100 million developers and counting**: There are now 100 million developers 💗 around the world using GitHub. [Here’s what this means](https://github.blog/2023-01-25-100-million-developers-and-counting/).
* **Introducing new validity checks for GitHub tokens**: GitHub now tells you whether GitHub tokens 🪙 found by secret scanning are active so you can prioritize and escalate remediation efforts. [Get the details](https://github.blog/2023-01-19-remediation-made-simple-introducing-new-validity-checks-for-github-tokens/). 

⚡**Beyond GitHub**: Open source content to help you level up! 
* **Simplicity is the ultimate sophistication**: [Hear about](https://github.com/readme/podcast/open-source-minimalism) 👂 how maximalism creeps into places where it doesn’t belong, finding mentors, following what energizes you, and the power of ping pong in the latest ReadME Project episode! 
* **Become a Git Wizard** 🧙‍♂️: Elliot Brenya sarfo [shares](https://dev.to/elliot_brenya/become-a-git-wizard-mastering-the-advanced-git-commands-3he4) how to cast more advanced commands. 

Wishing those who celebrate Lunar New Year a happy holiday 🏮!

**Comment by Samyylamba** [comment]
I love ❤️ GItHud but I only don’t know how to go about it 
and connect full,,

**Reply by Samyylamba** [reply]
![Uploading 1F52C29C-60E0-4D5C-A907-12C9129E9EB1.png…]()

**Reply by queenofcorgis** [reply]
Welcome to GitHub and the GitHub Community @samyylamba  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Comment by Maytham-MJ** [comment]
My advice to any web developer is get good at git.
